src/components/BitcoinConnectWallet.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <b-form-select v-model="selectedWallet" class="mb-3">
+      <b-form-select-option
+        :value="{walletName: 'Select your wallet', type: null}"
+        disabled
+      >Select your wallet</b-form-select-option>
+      <b-form-select-option :value="{walletName: 'Wasabi', type: 'p2p'}">Wasabi</b-form-select-option>
+      <b-form-select-option :value="{walletName: 'Electrum', type: 'electrum'}">Electrum</b-form-select-option>
+      <b-form-select-option :value="{walletName: 'BlueWallet (on-chain only)', type: 'electrum'}">BlueWallet</b-form-select-option>
+      <b-form-select-option :value="{walletName: 'Blockstream Green', type: 'p2p'}">Blockstream Green</b-form-select-option>
+      <b-form-select-option :value="{walletName: 'Phoenix', type: 'electrum'}">Phoenix</b-form-select-option>
+      <b-form-select-option-group label="Other">
+        <b-form-select-option
+          :value="{walletName: 'any other wallet that can connect to Bitcoin Core\'s P2P port', type: 'p2p'}"
```

Comment:
Can we change the wording a bit to avoid repeating the word "connect" twice:

```suggestion
:value="{walletName: 'any other wallet that supports Bitcoin Core\'s P2P port', type: 'p2p'}"
```

Reply:
Yeah, sounds much better.